src/test/java/org/folio/rest/impl/EholdingsTitlesTest.java

```diff
@@ -215,13 +229,4 @@ public void shouldReturnTitleWithoutResourcesWhenInvalidInclude() throws IOExcep
 
     JSONAssert.assertEquals(expected, actual, false);
   }
-
-  private ExtractableResponse<Response> getResponseWithStatus(String resourcePath, int expectedStatus) {
```

Comment:
""

Reply:
ok, never mind, I got it. Its been moved to WireMockTestBase.

Reply:
As above -- in base class